taskvine/src/manager/vine_manager.c

```diff
@@ -1174,32 +1331,15 @@ static int fetch_output_from_worker(struct vine_manager *q, struct vine_worker_i
 
 	// At this point, a task is completed.
 	reap_task_from_worker(q, w, t, VINE_TASK_RETRIEVED);
+	vine_manager_send(q, w, "kill %d\n", t->task_id);
 
-	switch (t->result) {
-	case VINE_RESULT_INPUT_MISSING:
-	case VINE_RESULT_FORSAKEN:
-		/* do not count tasks that didn't execute as complete, or finished tasks */
-		break;
-	default:
-		w->finished_tasks--;
-		w->total_tasks_complete++;
+	w->finished_tasks--;
```

Comment:
Is this function ever called for `FORSAKEN` tasks? If so, they should not be counted as finished tasks.

Reply:
Fixed to reflect previous handling.